.github/workflows/test-mlperf-inference-rgat.yml

```diff
@@ -10,6 +10,7 @@ on:
 
 jobs:
   build:
+    name: ${{ matrix.os }} - ${{ matrix.backend }} - ${{ matrix.implementation }}
```

Comment:
In the line above should we replace "build" with something better? May be "rgat-inference-run" ?